chord_metadata_service/mohpackets/tests/endpoints/test_overview.py

```diff
@@ -39,7 +39,7 @@ def setUp(self):
                 5,
                 program_id=self.programs[0],
                 gender=self.gender_man,
-                primary_site=[self.primary_site_sinus],
```

Comment:
Primary Site is now on the primary diagnosis -- where will this go?

Reply:
I forgot to put the test back in. I’ll fix it later, but it shouldn’t cause any problems for now.